app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,49 @@
+module SessionsHelper
+    # Logs in the given user.
+    def log_in(user)
```

Comment:
This whole file is Spaces: 4, please change it to Spaces: 2. 
If you do not understand how to apply this. Please ask in Chatwork.